tensorflow/BUILD

```diff
@@ -469,6 +469,7 @@ cc_binary(
     linkshared = 1,
     deps = [
         "//tensorflow/c:c_api",
+        "//tensorflow/c/eager:c_api",
```

Comment:
Thanks for this! I had no idea what the problem was. Is there a way for me to run the buildifier tool locally or is it not public?